src/core/common/ResourseTemplates/ListTemplate.js

```diff
@@ -146,6 +143,16 @@ class ListTemplate extends Component {
         })
     };
 
+    /**
+     * This function check is current page for list
+     *
+     * @author Bogdan Shcherban <bsc@piogroup.net>
+     * @return {boolean}
+     */
+    isListPage = () => {
+        return (this.props.location.pathname === "/" + this.props.resourceUrl);
```

Comment:
as a general rule, in modern JavaScript, prefer string interpolation to string concatenation.

    `/${this.props.resourceUrl}`

instead of 

    "/" + this.props.resourceUrl)

Reply:
Changed